wrappers/python/demo_writer.py

```diff
@@ -3,7 +3,9 @@
 from PIL import Image
 
 if len(sys.argv) < 3:
-	img = zxingcpp.write_barcode(zxingcpp.BarcodeFormat.QRCode, "I have the best words.", width=200, height=200)
+	img = zxingcpp.write_barcode(zxingcpp.BarcodeFormat.QRCode,
+		'I have the best words.', width=200, height=200)
 else:
-	img = zxingcpp.write_barcode(zxingcpp.barcode_format_from_str(sys.argv[1]), sys.argv[2], width=200, height=200)
-Image.fromarray(img).save("test.png")
+	img = zxingcpp.zxingcwrite_barcode(zxingcpp.barcode_format_from_str(sys.argv[1]),
```

Comment:
this code will not run, see above. Also as mentioned in earlier review: please revert every change except the `"` -> `'` replacement.